src/themes/creator.js

```diff
@@ -193,6 +193,10 @@ export default function createTheme(
       notificationNegativeBackground: primitives.negative50,
       notificationNegativeText: primitives.negative500,
 
+      // Table
+      tableHeadBackgroundColor: primitives.mono100,
+      tableHeadShadow: '0 1px 4px -1px hsla(0, 0%, 0%, 0.16)',
```

Comment:
Should it be `shadow400`?

Reply:
yes. fixed in f9181bfe924779b8c794716b079a65720c9b8035

Reply:
sorry, no... with `shadow400` it extended to the left/right when the shadow should only apply to bottom

Reply:
I didn't see that issue happening